README.md

```diff
@@ -1,2 +1,350 @@
+
+```bash
+deb http://ports.ubuntu.com/ubuntu-ports/ focal main restricted
+deb http://ports.ubuntu.com/ubuntu-ports/ focal-updates main restricted
+deb http://ports.ubuntu.com/ubuntu-ports/ focal universe
+deb http://ports.ubuntu.com/ubuntu-ports/ focal-updates universe
+deb http://ports.ubuntu.com/ubuntu-ports/ focal multiverse
+deb http://ports.ubuntu.com/ubuntu-ports/ focal-updates multiverse
+deb http://ports.ubuntu.com/ubuntu-ports/ focal-backports main restricted universe multiverse
+deb http://ports.ubuntu.com/ubuntu-ports/ focal-security main restricted
+deb http://ports.ubuntu.com/ubuntu-ports/ focal-security universe
+deb http://ports.ubuntu.com/ubuntu-ports/ focal-security multiverse
+```
+
+I had to mingle with these for a reason back then, but I do not remember when and why ... There is another file `sources.list.d` for any other files ending with `.list`, and they will be included in `sources.list` I guess. 
+
+
+Without straying from the main point anymore, let us go back to **`/opt`**. When you compile software that you have built, it can land here. **`/opt`** will divide itself structurally such as an application a library will be placed **`/opt/bin`** and **`/opt/lib`** respectively. When a package is installed in **`/opt`**, its static files should be in **`/opt/<package>`**. An important distinction is that there are other places for such software to be placed such as  **`/usr/local/bin`** and **`/usr/local/lib`**. This behavior is dictated by how the developers have configured files that control the compilation and installing process.
+
+---
+
+**`/proc`** contains information about your computer such as your CPU or the kernel. It is a virtual directory - it does not have an actual physical component, every physical disk is treated as a file in Linux (even the disk that you are working off of). that is why it is called virtual -, a pseudo-file system that the OS uses to (kernel \<-\> user space) communicate information about computer such as processes. The content is generated on the fly. When you `ps` for example, the OS gathers the process information from that directory. Below we see what is inside, note that the directories with numbers as their names are processes running and those numbers are their IDs or `pid`s.
+
+![listing the /proc directory content](/images/proc_dir.png)
+
+In this sense, we did not specify that but you may notice that **`/dev`** has the same feel to it as **`/proc`** too. And yes, it is a virtual directory too. **`/proc`** is a vital directory for us because we will get in here and establish our own cgroups and more. We will come to them later with utmost detail.
+
+---
+
+Let's check if this 'super user' guy is really super or a lamer with no life... 🧐
+
+![ls root](/images/lsa_root.png)
+
+**`/root`** is equivalent to `root` - a.k.a. administrator or super - users **`/home`**. 
```

Comment:
it seemed to me, these are the keywords that can help you to remember the topic if you are giving a presentation. Maybe you can elaborate a bit more :)
An outsider to the Linux filesystem probably won't understand what you mean here

Reply:
You mean `root` and `admin`? These are so engrained in life, I did not notice that I did not touch on them! Thank you, this is VERY precious.